### PR TITLE
style(progress-bars): canonize progress bars

### PR DESCRIPTION
**Pull request by eddywashere**
image

### CODE REVIEW

File: less/progress-bars.less [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.progress {
+  border: 1px solid #ccc;
+  border-radius: 0;
```
Comment (bradgignac) [inline less/progress-bars.less]: Can this rule be removed if you simply set the `@progress-border-radius` variable?

File: less/progress-bars.less [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  box-shadow: inset 0 0 0 1px #fff;
+  margin-bottom: 0;
```
Comment (bradgignac) [inline less/progress-bars.less]: Any reason we need to update this from the Bootstrap default?

Comment (eddywashere) [inline less/progress-bars.less]: negative. just updated.

**Comment by eddywashere** [comment]
@bradgignac updated

File: less/progress-bars.less [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.progress {
+  border: 1px solid #ccc;
+  border-radius: @progress-border-radius;
```
Comment (bradgignac) [inline less/progress-bars.less]: Is this already set by the Bootstrap source?

**Comment by bradgignac** [comment]
One question. LGTM otherwise.

**Pull request merged**